server/__init__.py

```diff
@@ -23,7 +23,7 @@
 from flask import Flask, request, g
 from flask_babel import Babel
 
-import en_core_web_lg
+import en_core_web_md
```

Comment:
What's the reason we're reducing from lg -> md?

Reply:
Ah, I felt the performance for place detection was similar/same but the size of the model is about 10x smaller :) Not sure why this came up in this PR because I think this was addressed in a previous PR. Sorry!